src/emucore/m6502/src/M6502.cxx

```diff
@@ -81,6 +82,16 @@ void M6502::reset()
   SP = 0xff;
   PS(0x20);
 
+    if (getenv("ALE_RANDOMIZE_CPU")) {
```

Comment:
OK I will make a separate PR for dev regarding randomization. From my functionality point of view I think the code is OK, but I dont like the way it is currently enabled (i.e. setting environment variable ALE_RANDOMIZE_CPU). What would be a clean way to do that?